@xen-orchestra/web/src/components/vm/VmVirtualisationAndBoot.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+  <UiCard class="vm-virtualisation-and-boot">
+    <UiTitle>
+      {{ $t('virtualization-boot-settings') }}
+    </UiTitle>
+    <VtsQuickInfoRow :label="$t('virtualization-mode')" :value="vm.virtualizationMode" />
+    <VtsQuickInfoRow :label="$t('secure-boot')">
+      <template #value>
+        <UiInfo :accent="vm.secureBoot ? 'success' : 'muted'">
+          {{ vm.secureBoot ? $t('enabled') : $t('disabled') }}
+        </UiInfo>
+      </template>
+    </VtsQuickInfoRow>
+    <VtsQuickInfoRow :label="$t('virtual-tpm')" :value="vm.VTPMs.join(', ')" />
+    <VtsQuickInfoRow :label="$t('viridian')">
+      <template #value>
+        <UiInfo :accent="vm.viridian ? 'success' : 'muted'">
+          {{ vm.viridian ? $t('enabled') : $t('disabled') }}
+        </UiInfo>
+      </template>
+    </VtsQuickInfoRow>
+    <VtsQuickInfoRow :label="$t('manage-citrix-pv-drivers-via-windows-update')" class="text-ellipsis">
+      <template #value>
+        <UiInfo accent="warning">
+          {{ $t('error-no-data') }}
+        </UiInfo>
+      </template>
```

Comment:
For this section, after checking with the team, the property to get from the API is `hasVendorDevice` (and `has_vendor_device` for XO Lite):
```suggestion
      <template #value>
        <UiInfo :accent="vm.hasVendorDevice ? 'success' : 'muted'">
          {{ vm.hasVendorDevice ? $t('enabled') : $t('disabled') }}
        </UiInfo>
      </template>
```
Remember to add the field in the API call.